### ISSUE TITLE
Model Transparency

### ISSUE THREAD

**Issue by jkennedyvz** [redhuntlabs/Octopii#3]
Is there a model card for the released checkpoint?

What dataset was used? Is it just the dummy-pii images, or were actual documents used for training?

https://github.com/redhuntlabs/Octopii/commit/8e3e749d787929df9045d141768296a1c05f7ffd#diff-ae44930471bf1a3aabba6def88e59bc916843fa5c25090a3898cebfe1e70423d

**Comment by owais-redhunt** [comment]
Hello. Octopii was created when we discovered leaky PII on some sites. When I was testing my proof of concept, I was testing it on some of the PII we discovered. I accidentally committed that PII to a private repo and forgot to erase git history before making it public.

I recently cleared this repo's history and emailed support to get rid of dangling commits as well. In the interest of these individuals' privacy, I will be request GitHub support to delete this issue.

Thanks for the heads up!
